hydra/app/jobs/generate_image_thumbs_job.rb

```diff
@@ -3,42 +3,25 @@ class GenerateImageThumbsJob < ApplicationJob
 
   queue_as :import
 
-  def perform(identifier)
+  def perform(id, download_path)
```

Comment:
Why the change from `id` to `identifier`?  Also in changing the method signature there could be impacts for enqueued jobs that are present between deploys.

Changing the name of the parameter from `id` to `identifier` created lots of little changes in the method.

Reply:
i thought changing would be better because i'm not passing in an identifier, i'm passing in an id, should i keep calling it an identifier even though it's an id?

Reply:
i added a comment in the model where the thumbnail jobs get called, that should clear up the confusion that this is an ActiveFedora id and not an identifier